lib/tasks/templates_rendering_statuses.rake

```diff
@@ -0,0 +1,24 @@
+# TRANSLATORS: do not translate
+desc <<~END_DESC
+  Refresh templates rendering statuses for all managed hosts
+
+    Example:
+      rake templates_rendering_statuses:refresh
+
+END_DESC
+
+namespace :templates_rendering_statuses do
+  task refresh: :environment do
+    Host::Managed.where(managed: true).in_batches(of: 100) do |batch|
```

Comment:
This block needs to run as `User.as_anonymous_admin` so the rake task can find all the templates.